dags/github.py

```diff
@@ -181,6 +182,25 @@ def extract_pull_requests(data):
         new_data = {"prs": prs, **data}
         return new_data
 
+    @task
+    def extract_pull_request_linked_issues(data):
+        logging.info(f"All data from last stage: {data}")
```

Comment:
I think let's find a way to manage the logs as we discussed before. if I remove them and we see errors in the future it would be hard to track the issue.

Reply:
Ok, agreed. but we need to remove this logs for production use. So please have in mind to fix these in production.